api/openapi.yaml

```diff
@@ -1442,52 +1550,32 @@ components:
           type: string
           format: date-time
           example: "2023-01-01T00:00:00Z"
-    LedgerEntryType:
+    RecurringPeriod:
```

Comment:
This needs to have more of a reference as well. 
Like every day at 00:00 UTC. Or do we expect it's always startOf ?

Reply:
We might have oversimplified with "fix 12am on X day", we can also introduce an anchor time as you suggested.